components/integration/production/stone-prod-p01/pipelines-as-code-secret-path.yaml

```diff
@@ -0,0 +1,4 @@
+---
+- op: add
+  path: /spec/dataFrom/0/extract/key
+  value: production/build/stone-prod-p01/github-app
```

Comment:
To be consistent, I think we can just use the vault key of pipeline-service `staging/pipeline-service/stone-stage-p01/github-app`(stage) and `production/pipeline-service/stone-prod-p01/github-app`(prod)  since we have already used the key of pipeline-service in https://github.com/redhat-appstudio/infra-deployments/blob/main/components/integration/base/external-secrets/pipelines-as-code-secret.yaml. Others look good to me.

Reply:
awesome, 
now I can see the logic ,
fixed